llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -18306,7 +18307,9 @@ NodeExtensionHelper::getSupportedFoldings(const SDNode *Root) {
   case RISCVISD::VFNMADD_VL:
   case RISCVISD::VFNMSUB_VL:
     Strategies.push_back(canFoldToVWWithSameExtension);
-    if (Root->getOpcode() == RISCVISD::VFMADD_VL)
+    // FIXME: Once other widen operations are supported we can merge
```

Comment:
hasStdExtZvfbfwma() doesn't have widening versions of VFNMADD_VL, VFNMSUB_VL, or VFMSUB_VL. What prevents generating those now?

Reply:
We need a test